ironfish-cli/src/commands/accounts/notes.ts

```diff
@@ -45,6 +45,9 @@ export class NotesCommand extends IronfishCommand {
             // Maximum memo length is 32 bytes
             minWidth: 33,
           },
+          sender: {
+            header: 'Sender Address',
```

Comment:
I think this can just say Sender, it's clear it's an address since you only send / to addresses.

Reply:
already merged, resolving here:
https://github.com/iron-fish/ironfish/pull/2764